website/content/docs/concepts/stateful-deployments.mdx

```diff
@@ -0,0 +1,57 @@
+---
+layout: docs
+page_title: Stateful Deployments
+description: |-
+  Learn how Nomad handles Stateful Deployments.
+---
+
+# Stateful Deployments
+
+<Note>
+Stateful deployments support only dynamic host volumes. For CSI volumes, we have
+<a href="/nomad/docs/job-specification/volume#per_alloc">per_alloc</a> property
+that serves a similar purpose.
+</Note>
```

Comment:
```suggestion
<Note>

Stateful deployments support only dynamic host volumes. For CSI volumes, use the
[`per_alloc` property](/nomad/docs/job-specification/volume#per_alloc), 
which serves a similar purpose.

</Note>
```

You can use markdown in a `<Note>` if you put a blank space before and after the note content. 

Replaced "we have" with "use" -> tell the reader what to use

Reply:
oh that's cool, I didn't know this about spacing+markdown.